salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -1,5 +1,9 @@
 {%- set dest_version = pillar.orchestrate.dest_version %}
 
+Refresh the package repository:
+  module.run:
+    - name: pkg.refresh_db
```

Comment:
No, you need to run it on the minion and just after the "Upgrade precheck".

Use `salt.function` instead of `module.run` and don't know if `pkg.refresh_db it's enough to fix the issue

Reply:
Wait no I think you should run it minion by minion in the for loop